### ISSUE TITLE
Is the WAKE_LOCK permission needed?

### ISSUE THREAD

**Issue by sgtpep** [termux/termux-boot#30]
Because of it `com.termux.boot` creates the wake lock which AFAIK blocks a device from sleeping. This is how it looks like:
```console
$ adb shell dumpsys power | grep -A 1 'Wake Locks'
Wake Locks: size=1
  PARTIAL_WAKE_LOCK              '*job*/com.termux.boot/.BootJobService' (uid=1000, pid=883, ws=WorkSource{10091})
```

Is it really needed, if a person already can control a wake lock using `termux-wake-{lock,unlock}`?

**Comment by sgtpep** [comment]
For that reason I managed to replace termux-boot with https://github.com/ITVlab/Launch-On-Boot, which doesn't require the WAKE_LOCK permission and launches termux which executes code from `~/.profile`.

**Comment by fornwall** [comment]
This should be fixed in version 0.6 of the app - please let us know otherwise!
